tiatoolbox/wsicore/metadata/ngff.py

```diff
@@ -0,0 +1,274 @@
+"""Generation of metadata for the OME-NGFF (zarr) slides.
```

Comment:
Do we need to import this in `metadata/__init__.py`?

Reply:
I don't expect it to be used outside of the ready function but could do.